lib/media/ttml_text_parser.js

```diff
@@ -217,6 +208,25 @@ shaka.media.TtmlTextParser.getLeafNodes_ = function(element) {
 
 
 /**
+ * Insert \n where <br> tags are found
+ *
+ * @param {!Node} element
```

Comment:
No problem.  I'm sorry that the compiler is a bit of a burden in this instance.  It's not perfect by any means, but I think on balance, it has helped to catch and prevent a lot of bugs.